Update peek.md (#5168)
MicrosoftDocs/windows-dev-docs: hub/powertoys/peek.md

---
title: PowerToys Peek utility for Windows
description: A system-wide utility for Windows to quickly preview file content.
ms.date: 11/19/2024

> ms.topic: concept-article

Reviewer: Please keep this new ms.topic type. It's one of our content patterns supported by some new tooling.

> ms.date: 08/03/2023

Reviewer: The ms.date should be updated to something more current.

Author: Ah. Sorry about this, the ms.topic and Customer Intent changes. This was because I edited the page as it was back in October when the original PR was raised. I will restore them in a new commit.

ms.date: 08/03/2023
ms.topic: article
no-loc: [PowerToys, Windows, Peek, Win]

> # Customer intent: Learn how to configure and use the Peek utility in PowerToys.

Reviewer: Please keep the "Customer intent". We're adding this for all content that uses one of the supported content pattern types.

## Pin preview window position and size

> The Peek window adjusts its size based on the dimensions of the images being previewed. However, if you prefer to keep the window's size and position, you can use the pinning feature.

Reviewer: Minor. If this is to be separate paragraphs, there must be an extra space between them or they'll be combined without any line feed.
```suggestion
The Peek window adjusts its size based on the dimensions of the images being previewed. However, if you prefer to keep the window's size and position, you can use the pinning feature.

```

By selecting the pinning button, the window will preserve the current size and position. Selecting the pinning button again will unpin the window. When unpinned, the Peek window will return to the default position and size when previewing the next file.

## Open file with the default program

Select **Open with** or <kbd>Enter</kbd> to open the current file with the default program.

## See extra information about the current file

Hover over the preview to see extra information about the file, including its size, type, and when it was last modified.

## Delete files

Press the <kbd>Delete</kbd> key to move the current file to the Recycle Bin.

By default, a confirmation dialog will appear before deletions. To skip future confirmations, either:

- Check the "Don't ask me again" checkbox in the dialog.
- Uncheck the "Ask for confirmation before deleting files" option in Peek's settings page.

> ![Example of Peek's file deletion confirmation dialog](../images/pt-peek-delete-confirmation.png)

Reviewer: Minor suggestion. I try to begin alt text with "A screenshot of" or "A picture of", etc:
```suggestion
![A screenshot of Peek's file deletion confirmation dialog](../images/pt-peek-delete-confirmation.png)
```

Select **Open with** or <kbd>Enter</kbd> to open the current file with the default program.

## See extra information about the current file

Hover over the preview to see extra information about the file, including its size, type, and when it was last modified.

## Delete files

Press the <kbd>Delete</kbd> key to send the current file to the Recycle Bin. Peek will automatically preview the next file. If there are no more files to preview, a message will be displayed.

> Note: only files may be deleted. Folders may not be deleted, even if they are empty.

Reviewer: ```suggestion
> [!TIP]
> Only files may be deleted. Folders may not be deleted, even if they are empty.
```
